python/ribasim/ribasim/schemas.py

```diff
@@ -1,37 +1,299 @@
-"""
-Generate Pandera Schemas from all autogenerated Pydantic Models
+# Automatically generated file.
+# DO NOT MODIFY.
 
-These classes have Schema as a postfix, so Ribasim.models.PumpStatic
-becomes Ribasim.schemas.PumpStaticSchema.
-"""
-import inspect
-import sys
+from datetime import datetime
 
 import pandera as pa
-from pandera.engines.pandas_engine import PydanticModel
-
-from ribasim import models
-
-
-def gen_schema(name, cls):
-    cname = f"{name}Schema"
-    ctype = type(
-        cname,
-        (pa.DataFrameModel,),
-        {
-            "Config": type(
-                f"{cname}.Config",
-                (),
-                {
-                    "dtype": PydanticModel(cls),
-                    "coerce": True,
-                    "add_missing_columns": True,
-                },
-            )
-        },
-    )
-    setattr(sys.modules[__name__], cname, ctype)
-
-
-for name, cls in inspect.getmembers(models, inspect.isclass):
-    gen_schema(name, cls)
+from pandera.typing import Series
+
+
+class BasinProfileSchema(pa.DataFrameModel):
+    node_id: Series[int] = pa.Field(nullable=False)
+    area: Series[float] = pa.Field(nullable=False)
+    level: Series[float] = pa.Field(nullable=False)
+
+    class Config:
+        add_missing_columns = True
+        coerce = True
```

Comment:
Perhaps we could simplify all these configs into one by creating our own base class.